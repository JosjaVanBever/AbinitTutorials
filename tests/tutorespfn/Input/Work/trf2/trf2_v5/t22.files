t22.in
t22.out
trf2_3.ddb.out
t22_dummy
t22_dummy1
t22_dummy2
t22_dummy3
