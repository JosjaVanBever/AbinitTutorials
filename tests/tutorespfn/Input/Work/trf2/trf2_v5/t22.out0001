
.Version 8.10.3 of ANADDB 
.(MPI version, prepared for a x86_64_linux_gnu5.4 computer) 

.Copyright (C) 1998-2018 ABINIT group . 
 ANADDB comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read https://docs.abinit.org/theory/acknowledgments for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Thu 31 Oct 2019.
- ( at 14h43 )
  

================================================================================

 -outvars_anaddb: echo values of input variables ----------------------

 Flags :
     dieflag         1
     ifcflag         1
     thmflag         1
 Miscellaneous information :
         asr         1
      chneut         2
 Frequency information :
       nfreq         1
       frmin         0.00000000E+00
       frmax         1.00000000E+01
 Interatomic Force Constants Inputs :
      dipdip         1
      ifcana         1
      ifcout        16
      natifc         2
       atifc         1         4
 Description of grid 1 :
        brav         4
       ngqpt         2         2         2
      nqshft         1
      q1shft
                     0.00000000E+00  0.00000000E+00  5.00000000E-01
 Thermal information :
       nchan      1250
      nwchan         5
      dostol         2.50000000E-01
      thmtol         3.00000000E-02
     ntemper        10
   temperinc         1.00000000E+02
   tempermin         1.00000000E+02
 Description of grid 2 (Fourier interp. or BZ sampling):
      ng2qpt        20        20        20
      ngrids         1
      q2shft         0.00000000E+00  0.00000000E+00  5.00000000E-01
 First list of wavevector (reduced coord.) :
       nph1l        11
       qph1l
                     0.00000000E+00  0.00000000E+00  0.00000000E+00    1.000E+00
                     0.00000000E+00  0.00000000E+00  5.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  4.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  4.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  3.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  3.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  2.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  2.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  1.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  1.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  5.00000000E-02    1.000E+00
 Second list of wavevector (cart. coord.) :
       nph2l         1
       qph2l
                     0.00000000E+00  0.00000000E+00  1.00000000E+00    0.000E+00

================================================================================

 read the DDB information and perform some checks

 Real(R)+Recip(G) space primitive vectors, cartesian coordinates (Bohr,Bohr^-1):
 R(1)=  0.0000000  5.3050000  5.3050000  G(1)= -0.0942507  0.0942507  0.0942507
 R(2)=  5.3050000  0.0000000  5.3050000  G(2)=  0.0942507 -0.0942507  0.0942507
 R(3)=  5.3050000  5.3050000  0.0000000  G(3)=  0.0942507  0.0942507 -0.0942507
 Unit cell volume ucvol=  2.9859750E+02 bohr^3
 Angles (23,13,12)=  6.00000000E+01  6.00000000E+01  6.00000000E+01 degrees
