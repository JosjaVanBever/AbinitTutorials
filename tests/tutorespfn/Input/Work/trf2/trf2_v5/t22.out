
.Version 8.10.3 of ANADDB 
.(MPI version, prepared for a x86_64_linux_gnu5.4 computer) 

.Copyright (C) 1998-2018 ABINIT group . 
 ANADDB comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read https://docs.abinit.org/theory/acknowledgments for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Thu 31 Oct 2019.
- ( at 14h45 )
  

================================================================================

 -outvars_anaddb: echo values of input variables ----------------------

 Flags :
     dieflag         1
     ifcflag         1
     thmflag         1
 Miscellaneous information :
         asr         1
      chneut         2
 Frequency information :
       nfreq         1
       frmin         0.00000000E+00
       frmax         1.00000000E+01
 Interatomic Force Constants Inputs :
      dipdip         1
      ifcana         1
      ifcout        16
      natifc         2
       atifc         1         2
 Description of grid 1 :
        brav         4
       ngqpt         2         2         2
      nqshft         1
      q1shft
                     0.00000000E+00  0.00000000E+00  5.00000000E-01
 Thermal information :
       nchan      1250
      nwchan         5
      dostol         2.50000000E-01
      thmtol         3.00000000E-02
     ntemper        10
   temperinc         1.00000000E+02
   tempermin         1.00000000E+02
 Description of grid 2 (Fourier interp. or BZ sampling):
      ng2qpt        20        20        20
      ngrids         1
      q2shft         0.00000000E+00  0.00000000E+00  5.00000000E-01
 First list of wavevector (reduced coord.) :
       nph1l        11
       qph1l
                     0.00000000E+00  0.00000000E+00  0.00000000E+00    1.000E+00
                     0.00000000E+00  0.00000000E+00  5.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  4.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  4.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  3.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  3.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  2.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  2.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  1.50000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  1.00000000E-01    1.000E+00
                     0.00000000E+00  0.00000000E+00  5.00000000E-02    1.000E+00
 Second list of wavevector (cart. coord.) :
       nph2l         1
       qph2l
                     0.00000000E+00  0.00000000E+00  1.00000000E+00    0.000E+00

================================================================================

 read the DDB information and perform some checks

 Real(R)+Recip(G) space primitive vectors, cartesian coordinates (Bohr,Bohr^-1):
 R(1)=  0.0000000  5.3050000  5.3050000  G(1)= -0.0942507  0.0942507  0.0942507
 R(2)=  5.3050000  0.0000000  5.3050000  G(2)=  0.0942507 -0.0942507  0.0942507
 R(3)=  5.3050000  5.3050000  0.0000000  G(3)=  0.0942507  0.0942507 -0.0942507
 Unit cell volume ucvol=  2.9859750E+02 bohr^3
 Angles (23,13,12)=  6.00000000E+01  6.00000000E+01  6.00000000E+01 degrees
 Now the whole DDB is in central memory

================================================================================

 Dielectric Tensor and Effective Charges 


 anaddb : Zero the imaginary part of the Dynamical Matrix at Gamma,
   and impose the ASR on the effective charges 

 The violation of the charge neutrality conditions
 by the effective charges is as follows :
    atom        electric field
 displacement     direction
       1               1       -0.022872        0.000000
       1               2        0.000000        0.000000
       1               3       -0.000000        0.000000
       2               1        0.000000        0.000000
       2               2       -0.022872        0.000000
       2               3       -0.000000        0.000000
       3               1        0.000000        0.000000
       3               2        0.000000        0.000000
       3               3       -0.022872        0.000000

 Effective charge tensors after 
 imposition of the charge neutrality,
 and eventual restriction to some part :
   atom    displacement
         1         1    2.106910E+00    0.000000E+00    7.656100E-20
         1         2    0.000000E+00    2.106910E+00    7.656100E-20
         1         3    0.000000E+00    0.000000E+00    2.106910E+00
         2         1   -2.106910E+00    0.000000E+00   -7.656100E-20
         2         2    0.000000E+00   -2.106910E+00   -7.656100E-20
         2         3    0.000000E+00    0.000000E+00   -2.106910E+00
 Now, the imaginary part of the dynamical matrix is zeroed

================================================================================

 Calculation of the interatomic forces 

-begin at tcpu      0.024  and twall      0.025 sec
